### PR TITLE
add put for custom alert. patch will not update SkipNotifications

### PR DESCRIPTION
**Pull request by jhanrahan-sigsci**

### CODE REVIEW

File: api.go [hunk 1]
```diff
@@ -573,12 +573,13 @@ func (sc *Client) ListCustomAlerts(corpName, siteName string) ([]CustomAlert, er
...
-	Threshold int    `json:"threshold"`
-	Enabled   bool   `json:"enabled"`
-	Action    string `json:"action"`
+	TagName           string `json:"tagName"`
+	LongName          string `json:"longName"`
+	Interval          int    `json:"interval"`
+	Threshold         int    `json:"threshold"`
+	Enabled           bool   `json:"enabled"`
+	Action            string `json:"action"`
+	SkipNotifications bool   `json:"skipNotifications"`
```
Comment (jhanrahan-sigsci) [inline api.go:582]: this changes the signature of a public struct

**Review by skhalsa-sigsci — APPROVED** [review]

**Review by mattg-sigsci — APPROVED** [review]
This seems fine, but it also seems like a bug that alert patch doesn't update SkipNotifications. Would it be better to just fix that instead?

**Comment by jhanrahan-sigsci** [comment]
Agreed, it does sound like a bug. I'll find someone on the api team and see if they know about it and are willing to fix it

**Comment by mattg-sigsci** [comment]
@jhanrahan-sigsci Steve Bartell would be a good person to start with.

**Review by mattg-sigsci — APPROVED** [review]
looks good. 🐬

**Comment by jhanrahan-sigsci** [comment]
Oh, i removed it because the PATCH method works now. Should I add it back in? Im not sure who would want it now, but it doesn't hurt to put it in

**Comment by mattg-sigsci** [comment]
Oh no, I think it's better to not have it. Sorry, I got confused about which PR this is. Looks good!

**Pull request merged**